### PR TITLE
[EDU-3919] generate credentials pt

### PR DESCRIPTION
**Pull request by gabriel-azion**
Related issue: [EDU-3919](https://aziontech.atlassian.net/browse/EDU-3919)

[EDU-3919]: https://aziontech.atlassian.net/browse/EDU-3919?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/pt-br/pages/deploy-jornada/gerar-credenciais/gerar-credenciais.mdx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+--header 'Authorization: Token [TOKEN VALUE]' \
+--header 'Content-Type: application/json' \
+--data '{
+  "description": "credential to use with Orchestrator",
+  "name": "new credential",
+  "status": true
+}'
+```
+
+Substitua `[TOKEN VALUE]` pelo seu [token pessoal](/pt-br/documentacao/produtos/guias/como-acessar-o-rtm/) e informe as seguintes propriedades no corpo da requisição:
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/deploy-jornada/gerar-credenciais/gerar-credenciais.mdx:52]:
```suggestion
Substitua `[TOKEN VALUE]` pelo seu [personal tokenl](/pt-br/documentacao/produtos/guias/como-acessar-o-rtm/) e informe as seguintes propriedades no corpo da requisição:
```

File: src/content/docs/pt-br/pages/deploy-jornada/gerar-credenciais/gerar-credenciais.mdx [hunk 2]
```diff
@@ -0,0 +1,71 @@
+---
+title: Como gerar credenciais
+description: Gere suas credenciais para a instalação do Edge Orchestrator Agent e começa a orquestrar seus serviços.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/deploy-jornada/gerar-credenciais/gerar-credenciais.mdx:3]:
```suggestion
description: Gere suas credenciais para a instalação do Edge Orchestrator Agent e comece a orquestrar seus serviços.
```

File: src/content/docs/pt-br/pages/deploy-jornada/gerar-credenciais/gerar-credenciais.mdx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+Para [instalar o Edge Orchestrator Agent](/documentacao/produtos/guias/deploy/instalar-orchestrator-agent/) e começar a orquestrar, antes você deve criar suas credenciais. Você pode fazer isso através de:
+
+- [Real-Time Manager (RTM)](/pt-br/documentacao/produtos/guias/como-acessar-o-rtm/)
+- [Azion API](https://api.azion.com/)
+
+---
+
+## Via RTM 
+
+1. Acesse [Real-Time Manager (RTM)](/pt-br/documentacao/produtos/guias/como-acessar-o-rtm/) > **Account menu**.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/deploy-jornada/gerar-credenciais/gerar-credenciais.mdx:19]:
```suggestion
1. Acesse [Real-Time Manager (RTM)](/pt-br/documentacao/produtos/guias/como-acessar-o-rtm/) > **Account menu** > **Credentials**.
```

Comment (MarianaAguilera) [reply src/content/docs/pt-br/pages/deploy-jornada/gerar-credenciais/gerar-credenciais.mdx:19]: revisar em EN

File: src/content/docs/pt-br/pages/deploy-jornada/gerar-credenciais/gerar-credenciais.mdx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+--header 'Authorization: Token [TOKEN VALUE]' \
+--header 'Content-Type: application/json' \
+--data '{
+  "description": "credential to use with Orchestrator",
+  "name": "new credential",
+  "status": true
+}'
+```
+
+Substitua `[TOKEN VALUE]` pelo seu [personal tokenl](/pt-br/documentacao/produtos/guias/como-acessar-o-rtm/) e informe as seguintes propriedades no corpo da requisição:
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/deploy-jornada/gerar-credenciais/gerar-credenciais.mdx:52]:
```suggestion
Substitua `[TOKEN VALUE]` pelo seu [personal tokenl](/pt-br/documentacao/produtos/guias/personal-tokens/) e informe as seguintes propriedades no corpo da requisição:
```

File: src/content/docs/pt-br/pages/deploy-jornada/gerar-credenciais/gerar-credenciais.mdx [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+| Propriedade | Descrição | Obrigatório | 
+| - | - | - | 
+| `name` | Nome da credencial que você deseja criar  | Sim |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/deploy-jornada/gerar-credenciais/gerar-credenciais.mdx:56]:
```suggestion
| `name` | Nome da credencial que você deseja criar | Sim |
```

File: src/content/docs/pt-br/pages/deploy-jornada/gerar-credenciais/gerar-credenciais.mdx [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+| `description` | Campo aberto para descrever sua credencial com informações como para que propósito ela foi criada | Não |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/deploy-jornada/gerar-credenciais/gerar-credenciais.mdx:57]:
```suggestion
| `description` | Campo aberto para descrever sua credencial com informações sobre seu propósito | Não |
```

File: src/content/docs/pt-br/pages/deploy-jornada/gerar-credenciais/gerar-credenciais.mdx [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+| `status` | Status da sua credencial | Não  |
+
+
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/deploy-jornada/gerar-credenciais/gerar-credenciais.mdx:60]:
```suggestion
```

File: src/content/docs/pt-br/pages/deploy-jornada/gerar-credenciais/gerar-credenciais.mdx [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+Agora você pode [instalar o Edge Orchestrator Agent](/documentacao/produtos/guias/deploy/instalar-orchestrator-agent/) no seu edge node e começar a orquestrar seus serviços.
+
+:::tip
+Confira a [documentação de API da Azion](https://api.azion.com/) e a [especificação OpenAPI](https://github.com/aziontech/azionapi-openapi/) para saber mais sobre todos os recursos disponíveis via API.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/deploy-jornada/gerar-credenciais/gerar-credenciais.mdx:64]:
```suggestion
Confira a [documentação da API da Azion](https://api.azion.com/) e a [especificação OpenAPI](https://github.com/aziontech/azionapi-openapi/) para saber mais sobre todos os recursos disponíveis via API.
```

**Review by MarianaAguilera — APPROVED** [review]

**Pull request merged**